### ISSUE TITLE
Upgrade dependency version

### ISSUE THREAD

**Issue by robinloh** [TanHaus/furni-mobile-app#29]
See https://github.com/TanHaus/furni-mobile-app/network/alert/package-lock.json/mem/open.

Check all `mem` versions. Upgrade them to ">=4.0.0".

**Comment by picasdan9** [comment]
Hi Robin, I did some investigation of this security bug.
I first ran `npm ls mem` in terminal to see the dependencies that `mem` might be nested in. This is the result.
```
`-- react-native@0.61.4
  +-- @react-native-community/cli@3.2.1
  | `-- metro@0.56.4
  |   `-- yargs@9.0.1
  |     `-- os-locale@2.1.0
  |       `-- mem@1.1.0
  `-- @react-native-community/cli-platform-android@3.1.4
    `-- logkitty@0.6.1
      `-- yargs@12.0.5
        `-- os-locale@3.1.0
          `-- mem@4.3.0
```
Some googling tells me that the `mem` vulnerability is fixed from `v4.9.0` of `@react-native-community/cli` (https://github.com/react-native-community/cli/issues/922).

However, I can't directly update `react-native` or `@react-native-community/cli` because currently the version of `react-native` is decided by expo SDK. Our project is already installed with the latest expo SDK.

I currently can't figure out a way to update a subdependency with command lines (I tried a few stackoverflow sites but none works). Another way is to modify the `package.json` and `pack-lock.json` codes (which might take some efforts referring to the source code of `@react-native-community/cli`), but I'm not sure if that will affect expo's behaviour afterwards.

**Comment by robinloh** [comment]
I'll close this issue for now.
